src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/TriageDataLocks/TriageDataLocksCommandHandler.cs

```diff
@@ -63,8 +63,12 @@ private static void Validate(TriageDataLocksCommand command, List<DataLockStatus
         {
             var courseAndPriceOrOnlyCourse = dataLocksToBeUpdated.All(DataLockStatusExtensions.WithCourseError)
                                           || dataLocksToBeUpdated.Any(DataLockStatusExtensions.WithCourseAndPriceError);
+
+            var IsResolved = apprenticeship.DataLockStatus.All(d => d.IsResolved && !d.IsExpired);
```

Comment:
not sure why we need this additional change here? If IsResolved = true, it won't be considered as a datalock to update.